src/elements/content-sharing/ContentSharing.js

```diff
@@ -151,28 +154,36 @@ function ContentSharing({ apiHost, displayInModal, itemID, itemType, language, t
         return <ErrorMask errorHeader={<FormattedMessage {...componentErrorMessage} />} />;
     }
 
-    if (item && sharedLink) {
+    // Ensure that all necessary data has been received before rendering child components
+    // "serverURL" is added to sharedLink after the call to the Users API
+    if (item && sharedLink && currentUserID && sharedLink.serverURL) {
```

Comment:
return null early to save some indentation?

Reply:
Good idea. Will refactor this in my next PR, which should have fewer changes than this one 🙂